__tasks__/build-riffraff-bundle.js

```diff
@@ -0,0 +1,104 @@
+// @flow
+
+const path = require('path');
+const cpy = require('cpy');
+const fs = require('fs');
+const { promisify } = require('util');
+const { warn, log } = require('./log');
+const exec = promisify(require('child_process').execFile);
+
+// This task generates the riff-raff bundle. It creates the following
+// directory layout under target/
+//
+// target/
+// ├── riffraff.yaml
+// ├── build.json
+// ├── guui/
+// │   └── dist/
+// │       └── guui.zip
+// ├── frontend-static/
+//     ├── javascripts/
+//     │   ├── ...js
+//     │   └── manifest.json
+//     └── stylesheets/
+//         └── ...css
+
+function copyCss(root = 'target') {
+    return cpy(
+        ['src/static/css/*'],
+        path.resolve(root, 'frontend-static', 'stylesheets'),
+    ).then(() => {
+        log('Finished copying css');
+    });
+}
+
+function copyJavascript(root = 'target') {
+    return cpy(
+        ['dist/*.js', 'dist/manifest.json'],
```

Comment:
i feel like needing to know how the riffraff deploy handles different file types that come out of bundling is going to bite us at some point. do we need to break them up (rather than just move all of dist)?

Reply:
How we arrange the files is entirely up to us really. Rifraff just takes our directory structure underneath target/frontend-static and transfers it to s3 where it's available on the assets.guim.co.uk domain. But we can arrange/name them however we want

Reply:
I guess the organisation under dist is something webpack handles - so if we can have webpack deciding how to organise dist, and have the script just copy all of dist, that could work too. We still have the static fonts css file which is only outside of dist as well though.